### DISCUSSION TITLE
Delayed exchange plugin testing dropwizard (Java)

### DISCUSSION THREAD

**Discussion by kkrhdd** [rabbitmq/rabbitmq-delayed-message-exchange#262] [Q&A] [answered]
How to make use of this plugin along with qpid broker for testing in Dropwizard framework (Java)?

**Comment by michaelklishin** [comment, marked as answer]
This is a RabbitMQ plugin, you cannot run it "along with qpid broker."

As for testing, publishing a message with a second delay, then testing how you would normally test routing with Dropwizard. RabbitMQ's own integration tests await a message delivery to a pre-defined queue, for example.
